packages/cra-template/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@fluentui/cra-template",
+  "version": "0.1.0",
+  "description": "CRA template for FluentUI Web",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui",
+    "directory": "packages/cra-template"
+  },
+  "scripts": {
+    "pretest": "npx create-react-app cra-test-app --template file:. && cd cra-test-app",
+    "test": "cd cra-test-app && SKIP_PREFLIGHT_CHECK=true yarn build && SKIP_PREFLIGHT_CHECK=true CI=true yarn test",
```

Comment:
Can you add this folder to the package npmignore and repo gitignore (in case deletion fails)? Or a better approach might be to make an orchestration script which puts this in a temp directory--there are a couple examples of this within the repo which I can help find if needed.

Reply:
I've now added a custom shell script which puts the app in a temp dir, and deletes it if everything was successful. Otherwise, the dir remains in place for potential further inspection. Lemme know if that's what you've had in mind!